packaging/rpm/microshift.spec

```diff
@@ -346,6 +368,9 @@ systemctl enable --now --quiet openvswitch || true
 %{_sysconfdir}/greenboot/red.d/40_microshift_pre_rollback.sh
 %{_datadir}/microshift/functions/greenboot.sh
 
+%files olm
```

Comment:
Also need this here to register a directory
```
%dir %{_prefix}/lib/microshift/manifests.d/001-microshift-olm
```

Reply:
done, PTAL